client/classic/api.js

```diff
@@ -0,0 +1,116 @@
+/* global wc */
+
+/**
+ * Internal dependencies
+ */
+import {
+	buildAjaxURL,
+	getConfig
+} from "../utils";
+
+/**
+ * Handles generic connections to the server and Stripe.
+ */
+export default class WCStripeAPI {
```

Comment:
Is there a reason we didn't add this to the `client/api` folder? 🤔 

I chose not to adapt the full WCPay API interaction code when I did the Blocks refactoring just to keep things simple, but even if we want to do that I still think this should live in `client/api` 🙂

Reply:
Good point... I didn't want to move the `stripe-utils` folder to the same level as `blocks`, `classic` and `api`.. but you are right that we should have only 1 API file, I'll change it